### ISSUE TITLE
Add feature to change a color of background for each layout (or maybe inherit it also to all layouts 

### ISSUE THREAD

**Issue by ask9** [wise-coders/dbschema#129]
I suggest to add feature to change a color of background for each layout (or maybe inherit it somehow also to all layouts of current dbs-file).
If it useful when you wor with different databases and the color mentally assigned with some database (project).
The color of background and its tab(!) helps user to manage layouts/projects better avoid confusing.
Here is a simple example of the idea:

![изображение](https://github.com/user-attachments/assets/2b0350c6-fbb5-402d-b62d-c712deacd8c1)

![изображение](https://github.com/user-attachments/assets/557e99f6-1ccd-4027-942f-310804e6e5d1)

1. Please check DbSchema Help / Output logs for errors.

2. Please include the DbSchema version, operating system and used database
DbSchema 9.6.0 build 240615
Windows 10
Postgres 16

3. The steps to reproduce this issue

**Comment by FredLackeyOfficial** [comment]
Agreed.  We need some way to alter the colors of the layouts.  They are simply too light to be beneficial and actually make it more difficult to see in some situations.

**Comment by dbschema-pro** [comment]
I think we can implement this. We will have a talk with our graphical designer.
